util/src/main/scala/com/scalableminds/util/tools/Math.scala

```diff
@@ -23,12 +23,18 @@ object Math {
     else
       c
   }
-  
+
   def roundDown(x: Double) = {
     val c = x.floor.toInt
     if(c != x)
       c - 1
     else
       c
   }
-}
\ No newline at end of file
+
+  def limit[T](x: T, lower: T, upper: T)(implicit num: Numeric[T]): T = {
```

Comment:
I haven't heard of `limit` for this usecase before and would have rather used `clamp`. But maybe I'm missing some street knowledge here :)